src/cmd-buildextend-azure

```diff
@@ -76,20 +76,23 @@ class Build(_Build):
         # See: https://docs.openstack.org/image-guide/convert-images.html
         run_verbose(['qemu-img', 'convert', '-f', 'qcow2', '-O', 'vpc',
                     tmp_img_azure, tmp_img_azure_vhd])
-        # Clean up the intermediate files
-        os.unlink(tmp_img_azure)
```

Comment:
Why not clean up the tmp qcow anymore? This is just the qcow with added platformid which we convert to the vhd. No other uses for it.

Reply:
Simple: the entire tmpdir gets whacked anyways :)

Reply:
Ah good point. LGTM!